main/plots.py

```diff
@@ -99,22 +101,225 @@ def create_capacity_planning_plot(start_date: datetime, end_date: datetime) -> f
         end_date: datetime object representing the end of the plotting period
 
     Returns:
-        Bokeh figure containing timeseries data.
+        Bokeh Row layout with timeseries data plot and CheckboxGroup widgets.
+
     """
-    effort_timeseries = timeseries.get_effort_timeseries(start_date, end_date)
+    # Get effort for each project and user
+    PROJECTS = ["proCAT 1", "proCAT 2", "proCAT 3", "proCAT 4"]
+    USERS = ["User 1", "User 2", "User 3", "User 4", "User 5"]
+    project_colours = ["firebrick", "orange", "green", "blue"]
+    user_colours = ["purple", "brown", "pink", "cyan", "magenta"]
+
+    # Total effort and capacity across all projects
+    total_effort_timeseries = timeseries.get_effort_timeseries(start_date, end_date)
     capacity_timeseries = timeseries.get_capacity_timeseries(start_date, end_date)
+
     traces = [
+        {"timeseries": capacity_timeseries, "colour": "navy", "label": "Capacity"},
         {
-            "timeseries": effort_timeseries,
-            "colour": "firebrick",
-            "label": "Project effort",
+            "timeseries": total_effort_timeseries,
+            "colour": "black",
+            "label": "Total effort",
         },
-        {"timeseries": capacity_timeseries, "colour": "navy", "label": "Capacity"},
     ]
+
+    # Add effort for each project
+    for i, project in enumerate(PROJECTS):
+        project_effort_timeseries = timeseries.get_project_effort_timeseries(
+            start_date, end_date, project
+        )
+        traces.append(
+            {
+                "timeseries": project_effort_timeseries,
+                "colour": project_colours[i],
+                "label": project,
+            }
+        )
+
+    # Add effort for each user
+    for i, user in enumerate(USERS):
+        user_effort_timeseries = timeseries.get_user_effort_timeseries(
+            start_date, end_date, user
+        )
+        traces.append(
+            {
+                "timeseries": user_effort_timeseries,
+                "colour": user_colours[i],
+                "label": user,
+            }
+        )
+
     plot = create_timeseries_plot(
         title="Project effort and team capacity over time", traces=traces
     )
-    return plot
+
+    plot.yaxis.axis_label = "Effort (hours)"
```

Comment:
The why axis, as it is now, is not hours, but "aggregated capacity", i.e. If 0.7 for you + 0.5 for me + ... etc. And might depend on the month.